CMakeLists.txt

```diff
@@ -86,6 +101,40 @@ message(STATUS "OpenCASCADE library directory: ${OpenCASCADE_LIBRARY_DIR}")
 include_directories(${OpenCASCADE_INCLUDE_DIR})
 link_directories(${OpenCASCADE_LIBRARY_DIR})
 
+# --------------------------------------------------------------------------- #
+# VTK
```

Comment:
so VTK is required now? but I think the IVtk* OCCT modules are skipped below. Is there a plan to make use of them?

Reply:
vtk seems to be installed by conda with occt, so it should always be there, but maybe this should be enabled with a flag anyways?  

Were those in the existing pyocct or are they new?  It was building fine but getting unresolved symbols when linking if I remember correctly.

Reply:
i don't see any vtk pyocct modules in the past. or even bindings generated for them. if the binder can process them, that's ok, but yes perhaps disable with a flag until we know how they can be used.